tests/local/Earthfile

```diff
@@ -137,7 +137,7 @@ test-copy-from-busybox-to-local-with-workdir:
     RUN cat /tmp/earthly-test-5816ebd4-6b18-421e-a1b4-5991af91e11e/0b59ac77-91d7-4b84-9d1c-12f64fc6f48e | grep "BusyBox v1.32.1"
 
 alpine-artifacts:
-    FROM alpine:3.15.0 # if this is changed, the grep in test-multi-copy-from-alpine-to-local must also be updated
+    FROM alpine:3.18.0 # if this is changed, the grep in test-multi-copy-from-alpine-to-local must also be updated
```

Comment:
should we use `3.18` instead of `3.18.0`?

Reply:
nope, we should use `3.18.3` and somehow instruct `find` and `sed` to read the comment in this file :rofl: 

I fixed the failing test in b0e7d3831875baf46972a1ca63213b4204bac54f